### PR TITLE
io500: update for SC'20 changes

### PR DESCRIPTION
**Pull request by adilger**
- remove the io-500-dev (bash script) checkout from prepare.sh
- remove running io500-scr from io500.sh
- remove parts of io500.sh that are no longer used
- update README-sc20.txt to remove references to bash version
- update .gitignore to skip temporary files

Signed-off-by: Andreas Dilger <adilger@dilger.ca>

### CODE REVIEW

File: README-sc20.txt [hunk 1]
```diff
@@ -0,0 +1,44 @@
+In the SC20 list, only the new C-application benchmark will be used.
+
+The io500.sh script can optionally be used to prepare the output
```
Comment (JulianKunkel) [inline README-sc20.txt:3]: I personally do not like artificial line breaks but have editors handle this as this allows everyone to enjoy line breaks at the width of their choice.
Also IMHO improves GIT merges to separate by sentence. But surely I'm OK with the choice you made. Just for discussion.

File: io500.sh [hunk 2]
```diff
@@ -2,43 +2,52 @@
...
+# Please visit https://vi4io.org/io500-info-creator/ to help generate the
+# "system-information.txt" file, by pasting the output of the info-creator.
+# This file contains details of your system hardware for your submission.
 
+# Set the paths to the binaries and how to launch MPI jobs.
+# If you ran ./prepare.sh successfully, then binaries are in ./bin/
 function setup_paths {
-  # Set the paths to the binaries and how to launch MPI jobs.
-  # If you ran ./utilities/prepare.sh successfully, then binaries are in ./bin/
   io500_ior_cmd=$PWD/bin/ior
```
Comment (JulianKunkel) [inline io500.sh:17]: We do not need this anymore as it is linked into the app directly.

**Review by JulianKunkel — APPROVED** [review]

**Review by johnbent — APPROVED** [review]
Looks great!

File: README-sc20.txt (hunk 1 above)
Comment (johnbent) [reply README-sc20.txt:3]: Yeah, when I used to program a lot, I got used to 80 char wide lines.  That's my typical preference.

File: README-sc20.txt [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+and logs for submission, or the io500 binary can be run directly.
+
+The test results will be put into a separate directory and tarred up for
+submission.  At a minimum, the io500.sh script needs the io500_mpirun and
+io500_mpiargs values set for your system in the setup_paths() function
+at the start.  The paths to the installed IOR and mdtest binaries are
+also needed if they were installed separately.  The script must be run
+from within the Git checkout tree.
+
+In detail, to setup the run:
```
Comment (johnbent) [inline README-sc20.txt:14]: I think we should make it clear that this is an optional step.

**Review by johnbent — APPROVED** [review]

File: io500.sh (hunk 2 above)
Comment (adilger) [reply io500.sh:17]: I guess the same is true for mdtest also, which is the whole point of the -app version...

**Comment by gflofst** [comment]
I approve

**Pull request merged**